Source/WebGPU/WebGPU/Queue.swift

```diff
@@ -28,11 +28,11 @@ private let largeBufferSize = 32 * 1024 * 1024
 public func writeBuffer(
     queue: WebGPU.Queue, buffer: WebGPU.Buffer, bufferOffset: UInt64, data: SpanUInt8
 ) {
-    queue.writeBuffer(mtlBuffer: buffer.buffer(), bufferOffset: bufferOffset, data: data)
+    queue.writeBuffer(buffer: buffer, bufferOffset: bufferOffset, data: data)
```

Comment:
@rr-codes fixed the `any` -> `some` here.
Actually I just removed the MTLBuffer and just passed around the WebGPU.Buffer instead.